src/main/java/org/zaproxy/zap/extension/hud/launch/ExtensionHUDlaunch.java

```diff
@@ -46,8 +50,15 @@
     private static final String SELENIUM_EXTENSION_CLASS_NAME =
             "org.zaproxy.zap.extension.selenium.ExtensionSelenium";
 
-    private CommandLineArgument[] arguments = new CommandLineArgument[1];
-    private static final int ARG_BROWSER_IDX = 0;
+    private CommandLineArgument[] arguments = new CommandLineArgument[3];
+    private static final int ARG_HUD_IDX = 0;
+    private static final int ARG_HUD_URL_IDX = 1;
+    private static final int ARG_HUD_BROWSER_IDX = 2;
+
+    private static final String FIREFOX = "Firefox";
+    private static final String CHROME = "Chrome";
+    private static final Set<String> SUPPOORTED_BROWSERS =
```

Comment:
Typo on support 😉

Reply:
ha!
Also just remembered that I'm not updated the help :(
Will update for both of those...

Reply:
Done both of those :)